daprdocs/content/en/developing-applications/ides/visual-studio-code/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Visual Studio Code integration with Dapr"
+linkTitle: "Visual Studio Code"
+weight: 1000
```

Comment:
Are the new topics that are broken out sub heading under a vscode index file? I could not see them listed.

Reply:
They are under vscode as my reasoning for the full name in the index file above. Once they were introduced to the full name I felt using the VSCode alias was more fitting now.

Reply:
Hugo lists this based on the location. When the site is built these are shown up correctly